### PR TITLE
Routing logic for Red Lake

### PR DESCRIPTION
**Pull request by andrewedstrom**
https://www.pivotaltracker.com/story/show/179456931

### CODE REVIEW

File: src/main/java/org/codeforamerica/shiba/pages/RoutingDecisionService.java [hunk 1]
```diff
@@ -66,6 +67,30 @@ public RoutingDecisionService(CountyParser countyParser,
     return List.of(countyRoutingDestinations.get(county));
   }
 
+  private List<RoutingDestination> routeRedLakeClients(Set<String> programs,
+      ApplicationData applicationData, Document document, County county) {
```
Comment (sree-cfa) [inline src/main/java/org/codeforamerica/shiba/pages/RoutingDecisionService.java:71]: Looks like `document` is an unused param here

File: src/main/java/org/codeforamerica/shiba/pages/RoutingDecisionService.java [hunk 2]
```diff
@@ -66,6 +67,30 @@ public RoutingDecisionService(CountyParser countyParser,
...
+
+    if (programs.contains(GRH)) {
+      return List.of(countyRoutingDestinations.get(county), tribalNations.get(RED_LAKE));
+    }
+
+    return List.of(tribalNations.get(RED_LAKE));
+  }
+
+  private boolean isLivingInTribalNationBoundary(ApplicationData applicationData) {
+    return Boolean.parseBoolean(
```
Comment (sree-cfa) [inline src/main/java/org/codeforamerica/shiba/pages/RoutingDecisionService.java:85]: Side thought - we could make this a convenient method in ApplicationDataParser "getBoolean(pagesData, field)"

**Comment by sree-cfa** [comment]
Looks good! The ticket has a "feature-flag" label but I didn't see anything flag related in this changeset - is that covered elsewhere?

**Comment by andrewedstrom** [comment]
I totally forgot about the feature flag piece of this. I'm not sure if the PMs actually want a feature flag or not, I'll follow up with them. In the meantime I'll merge this as is

**Pull request merged**